.github/workflows/test.yml

```diff
@@ -97,7 +97,7 @@ jobs:
     - name: Get pip cache dir
       id: pip-cache
       run: |
-        echo "::set-output name=dir::$(pip cache dir)"
+        echo "dir=$(pip cache dir)" >> $GITHUB_OUTPUT
```

Comment:
this change is for better readability?

Reply:
No, just following https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Reply:
Got it! Thanks for the heads up.